### ISSUE TITLE
ARG_VAR_MIXED reported when implicit struct has key equal to argument

### ISSUE THREAD

**Issue by rcastagno** [cflint/CFLint#478]
Scanning this should not give such error

```
<cfcomponent displayname="mycomponent">
    <cffunction name="myfunction" returntype="void">
    <cfargument name="myvariable" type="string" required="true">
        <cfscript>
            var mystructure = {myvariable = arguments.myvariable};
        </cfscript>
    </cffunction>
</cfcomponent>
```

Error goes away if I write either:

```
<cfcomponent displayname="mycomponent">
    <cffunction name="myfunction" returntype="void">
    <cfargument name="myvariable" type="string" required="true">
        <cfscript>
            var mystructure = {anothername = arguments.myvariable};
        </cfscript>
    </cffunction>
</cfcomponent>
```

or

```
<cfcomponent displayname="mycomponent">
    <cffunction name="myfunction" returntype="void">
    <cfargument name="myvariable" type="string" required="true">
        <cfscript>
            var mystructure = {myvariable = ''};
        </cfscript>
    </cffunction>
</cfcomponent>
```

Keys of implicit structs _should_ be between quotes, but as far as I know it's not required.

**Comment by ryaneberly** [comment]
fixed in dev branch.  Thanks @rcastagno
